src/OrleansCounterControl/Microsoft.Orleans.CounterControl.targets

```diff
@@ -1,4 +1,4 @@
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="15.0" DefaultTargets="Build">
```

Comment:
does it mean that this cannot be used by end users if they are still in Visual Studio 2015?